pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -1468,9 +1475,9 @@ public static List<KitRequestShipping> findKitRequest(@NonNull String field, @No
                         String realm = rs.getString(DBConstants.INSTANCE_NAME);
                         if (Arrays.asList(realms).contains(realm)) { //only if user has right to see that realm
                             String ddpParticipantId = rs.getString(DBConstants.DDP_PARTICIPANT_ID);
-                            String bspParticipantId = rs.getString(DBConstants.COLLABORATOR_PARTICIPANT_ID);
+                            String bspSampleId = rs.getString(DBConstants.BSP_COLLABORATOR_SAMPLE_ID);
```

Comment:
why are we changing this?

Reply:
because if participant has multiple samples then the key breaks